flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/ResultPartitionBuilder.java

```diff
@@ -50,6 +52,12 @@
 
     private NetworkBufferPool networkBufferPool = new NetworkBufferPool(2, 1);
 
+    private BatchShuffleReadBufferPool batchShuffleReadBufferPool =
+            new BatchShuffleReadBufferPool(1024, 32 * 1024);
```

Comment:
why set a `totalBytes` to be smaller than `bufferSize` in this default batchShuffleReadBufferPool? Seems it will lead to exceptions if used.